src/flavors/mississippi/config.json

```diff
@@ -0,0 +1,799 @@
+{
+  "flavor": {
+    "defaultLanguage": {
+      "code": "en",
+      "label": "English"
+    }
+  },
+  "datasets": [
+    {
+      "slug": "mississippi-input",
+      "url": "https://dev-api.mapseed.org/api/v2/smartercleanup/datasets/mississippi-input",
+      "clientSlug": "report",
+      "anonymous_permissions": [
+        {
+          "abilities": ["create"],
+          "submission_set": "*"
+        }
+      ]
+    }
+  ],
+  "app": {
+    "title": "an anthropocene river",
+    "name": "Mississippi",
+    "show_name_in_header": true,
+    "logo": "/static/css/images/Logo2.png",
+    "meta_description": "This map is dedicated to all the imhabitants of Cascadia, of every race and species",
+    "api_root": "https://dev-api.mapseed.org/api/v2/",
+    "theme": {
+      "brand": {
+        "primary": "#175f54",
+        "accent": "#2ebea8"
+      },
+      "bg": {
+        "default": "#175f54",
+        "light": "#000000"
+      },
+      "text": {
+        "tertiary": "#000000",
+        "titleFontFamily": "Roboto, sans-serif",
+        "titleColor": "#ffffff",
+        "headerFontFamily": "Roboto,sans-serif",
+        "bodyFontFamily": "Roboto,sans-serif"
+      }
+    }
+  },
+  "map": {
+    "geolocation_enabled": true,
+    "geolocation_onload": false,
+    "cartodb_enabled": true,
+    "geocoding_enabled": false,
+    "geocode_field_label": "Enter an address...",
+    "geocode_bounding_box": [],
+    "options": {
+      "mapViewport": {
+        "latitude": 39.77,
+        "longitude": -92.75,
+        "zoom": 5,
+        "pitch": 0,
+        "minZoom": 1,
+        "maxZoom": 18.4
+      }
+    },
+    "mapboxSources": {
+      "streets": {
+        "type": "raster",
+        "tiles": [
+          "https://api.mapbox.com/styles/v1/bkholmes/cjwk24ik402l81cot29vlnvsa/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoiYmtob2xtZXMiLCJhIjoiNjlkYjI4MDUyYTRlZWEyYzkwYTdmOTgxNmMzOGYwMTUifQ.VSUo52PYOUzS60NR6jqXTw"
+        ],
+        "tileSize": 256
+      },
+      "mississippi-watershed": {
+        "type": "raster",
+        "tiles": ["https://ecotopia.today/Mississippi/Basemap/{z}/{x}/{y}.png"],
+        "tileSize": 256
+      },
+      "oceans": {
+        "type": "raster",
+        "tiles": [
+          "https://services.arcgisonline.com/ArcGIS/rest/services/Ocean/World_Ocean_Base/MapServer/tile/{z}/{y}/{x}"
+        ],
+        "tileSize": 256
+      },
+      "satellite": {
+        "type": "raster",
+        "tiles": [
+          "https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"
+        ],
+        "tileSize": 256
+      },
+      "nitrogen": {
+        "type": "raster",
+        "tiles": [
+          "https://ecotopia.today/Mississippi/Nitrogen/{z}/{x}/{y}.png"
```

Comment:
I'm not sure why, but this layer isn't appearing. How would we debug this?

Reply:
I'm getting 404s on all the tile requests for this source, e.g.: https://ecotopia.today/Mississippi/Nitrogen/4/3/6.png.

Is the source info correct?